### ISSUE TITLE
Mac MIDI problem

### ISSUE THREAD

**Issue by ASBaumgarten** [LMMS/lmms#3748]
Recently dug up my old Novation Launchkey Mini (MK1) to try and get it to work with LMMS. Normally I relegate all my actual projects to my MacBook Pro, but I can't seem to get the controller to be recognized in LMMS. The controller connects to my system and registers inputs via [MIDI Monitor](https://www.snoize.com/MIDIMonitor/) and Audio MIDI Setup. Despite trying almost every configuration the controller won't show up as an option in the MIDI section of LMMS settings. It will only show me the 'Dummy' option in the list. The controller works on Windows just fine with the same LMMS 1.1.3.

```
Hardware Overview:

  Model Name:	MacBook Pro
  Model Identifier:	MacBookPro11,3
  Processor Name:	Intel Core i7
  Processor Speed:	2.3 GHz
  Number of Processors:	1
  Total Number of Cores:	4
  L2 Cache (per Core):	256 KB
  L3 Cache:	6 MB
  Memory:	16 GB

LMMS: Version 1.1.3
  
```

**Comment by Umcaruje** [comment]
LMMS 1.1.3 did not support MIDI on mac, this was implemented with #2228 and is available in the latest RC of LMMS 1.2

Closing as duplicate of #1153

**Comment by tresf** [comment]
> is available in the latest RC of LMMS 1.2

Specifically, https://lmms.io/download, scroll down to beta version.

**Comment by ASBaumgarten** [comment]
Got it. Works as intended.
